indico/modules/events/registration/controllers/api/checkin.py

```diff
@@ -107,3 +107,18 @@ def _process_PATCH(self, checked_in=None, paid=None):
         if paid is not None and paid != self.registration.is_paid and self.registration.price:
             toggle_registration_payment(self.registration, paid=paid)
         return CheckinRegistrationSchema().jsonify(self.registration)
+
+
+class RHCheckinAPIRegistrationUUID(RHCheckinAPIBase):
```

Comment:
Please cover this one in the `test_unauthorized` test. I'm pretty sure we get proper access checks via inheritance, but still better to have it in the tests.